### PR TITLE
update Makefile

### PR DESCRIPTION
**Pull request by terencehonles**
- add target "frontend_wheels" to build frontend wheels for testing somewhere "pip install -e ." is not appropriate.
- remove target "clean" as it is more dangerous than most would expect (it does not just clean built artifacts).

## Contributor Checklist:

* [ ] I have updated the unit tests
* [ ] I have created a file in the `master/buildbot/newsfragments` directory (and read the `README.txt` in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

**Comment by terencehonles** [comment]
See pre existing comments:

- https://github.com/buildbot/buildbot/pull/5551#discussion_r490487897
- https://github.com/buildbot/buildbot/pull/5551#discussion_r490488484
- https://github.com/buildbot/buildbot/pull/5551#discussion_r490489480

File: Makefile [hunk 1]
```diff
@@ -143,7 +150,9 @@ tarballs: $(ALL_PKGS_TARGETS)
 
 
 clean:
+	@if [ -t 0 ]; then read -p 'You are about to delete all untracked files! (use [Ctrl+C] to abort) '; fi
 	git clean -xdf
```
Comment (p12tic) [inline Makefile:154]: Maybe it makes sense to remove this rule altogether? It seems that the user is better off just calling `git clean` manually.

Comment (tardyp) [reply Makefile:154]: agreed.

**Review by tardyp — CHANGES_REQUESTED** [review]
lets remove make clean. +1 for the rest

File: Makefile (hunk 1 above)
Comment (terencehonles) [reply Makefile:154]: sounds good to me, wasn't sure if it was used anywhere

**Review by tardyp — APPROVED** [review]

**Pull request merged**